.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    name: 'Tagged Release'
+    runs-on: 'ubuntu-latest'
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: wyvox/action-setup-pnpm@v3
```

Comment:
We need to pass it the NPM token somehow. I'll add it as `secrets.NPM_TOKEN` and you can figure out how to get it to the right place?

Reply:
yup! -- added

Reply:
Added `NPM_TOKEN`